src/components/organisms/top/top.scss

```
@@ -0,0 +1,51 @@
+@use "../../variables" as *;
+
+.top {
+  .top-image {
+    max-width: 100%;
+    height: auto;
+    display: block;
+    position: relative;
+
+    img {
+      height: 100vh;
+      width: 100%;
+      object-fit: cover;
+      @media (max-width: $breakpoint-lg) {
+        height: 360px;
+      }
+    }
+
+    .text-overlay {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      gap: 10px;
+
+      p {
+        text-align: center;
+        font-size: $fontsize-xl;
+        color: $white;
+        writing-mode: vertical-rl;
+        text-align: start;
+        font-family: $fontfamily-mincho;
+
+        @media (max-width: $breakpoint-lg) {
+          font-size: $fontsize-l;
+        }
+      }
+    }
+  }
+  .message {
+    padding: 52px 20px;
+
+    @media (max-width: $breakpoint-lg) {
+      padding: 32px 20px;
+    }
+  }
+  #message1 {
+    color: $white;
+    background-color: $black;
+  }

```

```suggestion
  .message {
    padding: 52px 20px;
    color: $white;
    background-color: $black;

    @media (max-width: $breakpoint-lg) {
      padding: 32px 20px;
    }
  }
```
特に理由なければidセレクタは使わない形で書くのが望ましいです！
（詳細度を低く保つ観点）

---

なるほど！ここだけ修正してマージします！